http/src/main/resources/init-resources/proxy-docker-compose.yaml

```diff
@@ -2,8 +2,9 @@ version: '2.4'
 services:
   proxy:
     container_name: "${PROXY_SERVER_NAME}"
-    image: "${PROXY_DOCKER_IMAGE}"
+    image: "mirror.gcr.io/${PROXY_DOCKER_IMAGE}"
```

Comment:
Might as well leave the mirror in, even if dockerhub was not the culprit in the end